lib/src/source/git.dart

```diff
@@ -142,6 +142,8 @@ class GitSource extends Source {
     return {'relative': relative, 'url': url};
   }
 
+  /// Returns [path] normalized.
+  ///
```

Comment:
Also, say what a "relative URL" is. It's not clear from context, and not a well-defined term.

The `Uri.isAbsolute` checks for a scheme and no fragment, so `http://example.com/foo#bar` is *not* considered absolute (it has a fragment!)
It's very unlikely to ever be what you want.

You may want `uri.hasAbsolutePath` instead, and check that `url.hasScheme` and `url.hasAuthority` are both false. (Because `scheme:relative/path#fragment` is not `isAbsolute` and has a relative path, but is probably not what you'd consider a releative URI reference).

Reply:
Ah I thought a relative uri was any non-absolute one.

Thanks, will try to see if I can come up with something...